README.md

```diff
@@ -1,2 +1,70 @@
-# enhancements
-Enhancement Tracking and Backlog Repo for Istio Releases
+# Enhancement Tracking and Backlog
+Enhancement Tracking and Backlog Repo for Istio Releases. Owned by TOC.
+
+This repo contains issues and Istio enhancement proposal. These issues are umbrellas for new enhancements to be added to Istio. An enhancement usually takes multiple releases to complete. And an enhancement can be tracked as backlog items before work begins. An enhancement may be filed once there is consensus in at least one Istio working group.
```

Comment:
What's the difference between an issue and enhancement in this context?

Reply:
Are enhancement proposals github issues or files checked into the enhancement repo?

Reply:
issue is a github issue, where enhancement proposal is the proposal linked to the github issue.  I think there are preference that the proposal checked into github after approval.

Reply:
Commit proposal makes sense to me. The wording here seemed to imply the proposal would only exist as an issue, in which case I didn't see the value over tracking issues in istio/istio.